src/Controls/src/Core/Border.cs

```diff
@@ -7,14 +10,21 @@ namespace Microsoft.Maui.Controls
 	[ContentProperty("Content")]
 	public class Border : View, IContentView, IBorder, IPaddingElement
 	{
-		public static readonly BindableProperty ContentProperty = BindableProperty.Create(nameof(Content), typeof(IView),
+		ReadOnlyCollection<Element>? _logicalChildren;
```

Comment:
Border is an IContentView, so by definition it can only have one child (Content). Do we need a whole ObservableCollection for this? We could implement IVisualTreeElement by returning a ReadOnlyList that's either empty or has Content in it.

Reply:
I was using the base Element thing and overriding that. Not sure if it is "safe" to just re-implement the stuff. I see all our other things override and that returns a list.

I suppose I can return a dummy array with just content...

@PureWeen is it safe just to ignore the weird `internal override IReadOnlyList<Element> LogicalChildrenInternal`? I don't know this part well enough to have a vote on the best way.

Reply:
The `LogicalChildrenInternal` does a lot more to propagate styles, contexts and more.